gotocompany/shield/v1beta1/shield.proto

```diff
@@ -504,6 +504,28 @@ service ShieldService {
       summary: "Create Service Data Key";
     };
   }
+
+  rpc UpdateUserServiceData(UpdateUserServiceDataRequest) returns (UpdateUserServiceDataResponse) {
+    option (google.api.http) = {
+      put: "/v1beta1/users/{id}/servicedata",
+      body: "body"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      tags: "Service Data";
+      summary: "Update User Service Data";
+    };
+  }
+
+  rpc UpdateGroupServiceData(UpdateGroupServiceDataRequest) returns (UpdateGroupServiceDataResponse) {
+    option (google.api.http) = {
+      put: "/v1beta1/groups/{id}/servicedata",
+      body: "body"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      tags: "Service Data";
+      summary: "Update Group Service Data";
+    };
+  }
 }
```

Comment:
This should be exposed publicly and not part of the admin API.
Can we rename the existing file to `admin.proto` and create another proto called `shield.proto` where we put non-admin API in that?

Reply:
Sorry my bad, we don't need to rename it but we should create another proto file to put this exposed API and not clubbing it withing the same proto

Reply:
just realized this too. will do